docs/source/models.rst

```diff
@@ -486,13 +486,13 @@ Model                             Acc@1          Acc@5
 ================================  =============  =============
 MobileNet V2                      71.658         90.150
 MobileNet V3 Large                73.004         90.858
-ShuffleNet V2 x1.0                68.360         87.582
-ShuffleNet V2 x0.5                57.972         79.780
-ResNet 18                         69.494         88.882
-ResNet 50                         75.920         92.814
-ResNext 101 32x8d                 78.986         94.480
+ShuffleNet V2 x1.0                67.886         87.332
```

Comment:
These are pretty big accuracy gaps. Diffs smaller than 0.2 could be attributed to randomness but anything bigger needs to be investigated. Any thoughts on why we get differences of 0.5% here? Note that if this is indeed randomness, doing 2-3 more runs should give you similar good results.

I also noticed that you didn't update the values for MobileNet V2/V3 and Inception. What are the updated numbers for these models?

Reply:
Ok, I will rerun them again to see if we get a smaller gap. Also, these numbers are now outdated and I will follow up with the latest set of accuracies. For MobileNet, they are taking a long time to run so I don't have the numbers yet. For inception, I was not able to reproduce the original accuracies in my initial attempt but I am still working on it.

Reply:
Sounds good. Let me know if I can help with any of it.

Reply:
Just reran and uploaded the new numbers as part of the PR description. There was a lot of variance across runs so I just selected the runs with the highest accuracies. Note that inception and mobilenet are still TODOs at this point.

Reply:
Just reran and uploaded the new numbers as part of the PR description (see table). There was a lot of variance across runs so I just selected the runs with the highest accuracies. Everything should be within 0.2 or 0.3 of the reference accuracy now. As for mobilenet, I'm still waiting on the results.